x/accounts/defaults/lockup/utils_test.go

```diff
@@ -77,7 +77,9 @@ func newMockContext(t *testing.T) (context.Context, store.KVStoreService) {
 			case "/cosmos.staking.v1beta1.MsgDelegate":
 				return &stakingtypes.MsgDelegateResponse{}, nil
 			case "/cosmos.staking.v1beta1.MsgUndelegate":
-				return &stakingtypes.MsgUndelegate{}, nil
+				return &stakingtypes.MsgUndelegateResponse{
+					Amount: sdk.NewCoin("test", math.NewInt(1)),
```

Comment:
Should the denom be `stake` instead?

Reply:
ah since my mock context query method return bond denom as test so this is fine, maybe i can rename it from test to stake